### PR TITLE
[FIX] fixed wrong number of study variable columns

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

File: src/topp/MzTabExporter.cpp [hunk 1]
```diff
@@ -730,7 +730,7 @@ namespace OpenMS
 
       // currently we don't save how many channels we saved in a consensus feature so we scan the file
       // for the maximum max index in a consensus feature and take this as number of study variables
```
Comment (cbielow) [inline src/topp/MzTabExporter.cpp]: typo 'map' index

Comment (cbielow) [inline src/topp/MzTabExporter.cpp]: what is
https://github.com/timosachsenberg/OpenMS/blob/fix/MzTab_column_count/src/topp/MzTabExporter.cpp#L834
doing?

**Comment by cbielow** [comment]
+1

File: src/topp/MzTabExporter.cpp (hunk 1 above)
Comment (timosachsenberg) [inline src/topp/MzTabExporter.cpp]:
```
  row.peptide_abundance_stdev_study_variable[study_variable];
```
Calls the default constructor for the entry which sets this entry to "null" aka not set.

**Pull request merged**